src/main/java/org/sagebionetworks/web/client/jsinterop/EntityModalProps.java

```diff
@@ -19,23 +19,23 @@ public interface BooleanCallback {
     void run(boolean value);
   }
 
-  String entityId;
-  boolean show;
+  public String entityId;
+  public boolean show;
 
   @JsNullable
-  Callback onClose;
+  public Callback onClose;
 
   @JsNullable
-  String initialTab; // "METADATA" | "ANNOTATIONS"
+  public String initialTab; // "METADATA" | "ANNOTATIONS"
 
   @JsNullable
-  boolean showTabs;
+  public boolean showTabs;
 
   @JsNullable
-  Double versionNumber;
+  public Long versionNumber;
```

Comment:
I have no idea why this was ever `Double`. `Long` works and is more correct.

Reply:
Long has required special consideration in GWT.  Maybe the dev read this in the [compatibility notes](https://www.gwtproject.org/doc/latest/DevGuideCodingBasicsCompatibility): "Additionally, long primitives cannot be used in [JSNI code](https://www.gwtproject.org/doc/latest/DevGuideCodingBasics.html#DevGuideJavaScriptNativeInterface) because they are not a native JavaScript numeric type."